graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/site/GqlJcrSite.java

```diff
@@ -135,4 +135,11 @@ public String getDefaultLanguage() {
         return result;
     }
 
+    @GraphQLField
+    @GraphQLName("homePage")
+    @GraphQLDescription("Returns the node of the home page")
+    public GqlJcrNode getHomePage() throws RepositoryException {
+        return new GqlJcrNodeImpl(siteNode.getHome());
```

Comment:
you should not instantiate GqlJcrNodeImpl drectly, you can use `SpecializedTypeHandler.getNode`

Reply:
Thanks for the hint!